zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -34,6 +34,10 @@ def query_volume_info(logical_pool_name, lun_name):
     return shell.call("%s query file --path %s/%s --format json" % (DEFAULT_ZBS_BIN_PATH, logical_pool_name, lun_name))
 
 
+def query_volumes_in_logical_pool(logical_pool_name):
+    return shell.call("%s list file --pool %s --format json" % (DEFAULT_ZBS_BIN_PATH, logical_pool_name))
```

Ruff (0.8.2)

`shell` may be undefined, or defined from star imports

(F405)